gallery/tutorial/tvmc_python.py

```diff
@@ -111,7 +111,7 @@
 #   result = tvmc.run(package, device="cpu") #Step 3: Run
 #
 # And you can print the results:
-# ``print(results)``
+# ``print(result)``
```

Comment:
Should we also fix `my_model` in line 71? Like:

s/`my_model`/`'my_model.onnx'`/  ? 

Also, since you'll need to re-trigger the CI, would you mind to:

in lines 240 and 245 add a space after second comma, i.e.  s/`"llvm",tuning_records`/`"llvm", tuning_records`/ ?

Reply:
OK, I have re-triggered the CI and fix typos mentioned above.